src/test/java/page/googlecloud/GoogleCloudPricingCalculatorPage.java

```diff
@@ -0,0 +1,97 @@
+package page.googlecloud;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class GoogleCloudPricingCalculatorPage {
+
+	@FindBy(css = "[ng-model$='quantity']")
+	public WebElement inputNumberOfInstances;
+
+	@FindBy(css = "iframe[src*=calculator]")
+	public WebElement calculatorIFrame;
+
+	@FindBy(id = "myFrame")
+	public WebElement myFrame;
```

Comment:
I believe it's not the best name for the iframe
```suggestion
	public WebElement internalCalculatorFrame;
```